tests/test_recv.py

```
@@ -0,0 +1,172 @@
+		with mock.patch.object(
+			multicast.recv.McastRECV, '_hearstep', return_value='Test response'
+		):
+			self.recv_tool.doStep(is_std=True)
+			# Verify that the response is printed to console
+			output = mock_stdout.getvalue()
+			self.assertIn('Test response', output)
+			# Verify that the logger recorded the intended debug message
+			mock_logger.debug.assert_any_call("Will Print to Console.")
+
+	@mock.patch('multicast.recv.module_logger')
+	def test_doStep_with_custom_parameters(self, mock_logger):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_doStep_with_custom_parameters`

Add return type annotation: `None`

(ANN201)